### ISSUE TITLE
Request for adding a function: determining the start of card movement

### ISSUE THREAD

**Issue by wfgm5k2d** [ricardodalarme/flutter_card_swiper#27]
Hello! I really like your package flutter_card_swiper, it almost meets my needs. I would like to know how I can get the card offset coordinate? I'll take a picture. The onSwipe event shows the final direction after moving the card, as does the onSwipeDirectionChange event. I would like to be able to determine the beginning of the movement and the direction, perhaps get the X and Y coordinates. It is important for me to know the moment the movement begins, the direction in which direction the movement is carried out and the end of the movement when the card is flipped. Please tell me, is it possible to somehow find out this now or is there such functionality in the plans?

**Comment by ektapadaliya** [comment]
@wfgm5k2d Did you find any solution? 
I'm finding a way to tell that animation start.

**Comment by awaisabbasaa11** [comment]
@wfgm5k2d can you please share your solution? Thanks

**Comment by wfgm5k2d** [comment]
Certainly. You already have coordinates that are transferred to the builder.

body: CardSwiper(
  controller: cardSwiperController,
  onSwipe: _onSwipe,
  onUndo: _onUndo,
  onSwipeDirectionChange: _onSwipeDirectionChange,
  cardsCount: users.length,
  cardBuilder: (context, index, *percentThresholdX*,
*percentThresholdY*) => UserCardWidget(
    externalController: cardSwiperController,
    user: users[index], // Индекс в массиве пользователей
    *percentThresholdX: percentThresholdX**,*
    *percentThresholdY: percentThresholdY**,*
  ),
  scale: 0.9,
  padding: EdgeInsets.symmetric(horizontal: 16, vertical: 0),
),


сб, 15 июн. 2024 г. в 10:43, Awais Abbas ***@***.***>:

**Comment by awaisabbasaa11** [comment]
@wfgm5k2d Thank you for your quick reply i solved this by following your approach, much appreciated
